packages/ddp/livedata_connection.js

```diff
@@ -520,12 +532,41 @@ _.extend(Connection.prototype, {
         if (!existing.ready)
           existing.readyCallback = callbacks.onReady;
       }
+
+      // XXX COMPAT WITH 1.0.3.1 we used to have onError but now we call
+      // onStopped with an optional error argument
       if (callbacks.onError) {
         // Replace existing callback if any, so that errors aren't
         // double-reported.
         existing.errorCallback = callbacks.onError;
       }
+
+      if (callbacks.onStopped) {
+        existing.stoppedCallback = callbacks.onStopped;
+      }
+
+      // Return the exact same handle so that we can use === to check if the
```

Comment:
That makes sense, but in that case it should probably be documented, at least internally.  At the very least, the (good) comment that you added here could have a link to the blaze code that cares.  Or it could even be in the real docs (which does describe the de-duplication), though that links would still be good.

On the other hand --- the `id` is an actual real part of the DDP wire protocol, not an internal implementation detail of the ddp package, and arguably we should just expose it on the handle?